_includes/head.html

```diff
@@ -8,7 +8,4 @@
     {%- if jekyll.environment == 'production' and site.google_analytics -%}
     {%- include google-analytics.html -%}
     {%- endif -%}
-
-    <script type="text/javascript" src="https://api.mapy.cz/loader.js"></script>
```

Comment:
Když jsem to ladil, tak jsem trochu koukal do konzole a rovnou jsem vyhodil mapy.cz, ať se nenačítají všude. Potřeba jsou jen na detailu hornin.